jvm/advanced/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- https://github.com/redskyops/redskyops-recipes/jvm
+
+patchesJson6902:
+- target:
+    group: redskyops.dev
+    version: v1alpha1
```

Comment:
this probably needs to get changed to v1beta1

Reply:
Good catch!